docs/setup/ENVIRONMENT_MAPPING.md

```
@@ -0,0 +1,235 @@
+
+---
+
+## 📊 Supabaseプロジェクト情報
+
+### Production環境
+
+| 項目 | 値 |
+|------|-----|
+| **プロジェクト名** | ppt-trans-production |
+| **Project ID** | gjiijbmyterxmggryvcb |
+| **URL** | https://gjiijbmyterxmggryvcb.supabase.co |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)